src/Support/helpers.php

```diff
@@ -317,3 +317,15 @@ function http_build_url(array $url, array $replace = [], $flags = HTTP_URL_REPLA
         return \October\Rain\Router\UrlGenerator::buildUrl($url, $replace, $flags, $newUrl);
     }
 }
+
+if (!function_exists('is_countable')) {
+    /**
+     * Polyfill for `is_countable` method provided in PHP 7.3
+     *
+     * @param  mixed  $var
+     * @return boolean
+     */
+    function is_countable($value) {
```

Comment:
@Samuell1 Please put the opening brace on its own line.
```suggestion
    function is_countable($value)
    {
```